build.gradle

```diff
@@ -36,9 +36,9 @@ repositories {
 
 dependencies {
 
-	implementation 'uk.gov.laa.ccms.data:data-api:0.0.16'
+	implementation 'uk.gov.laa.ccms.data:data-api:0.0.20'
 
-	implementation 'uk.gov.laa.ccms.soa.gateway:soa-gateway-api:0.0.35'
+	implementation 'uk.gov.laa.ccms.soa.gateway:soa-gateway-api:0.0.36-4f8fbb6-SNAPSHOT'
```

Comment:
To be updated once https://github.com/ministryofjustice/laa-ccms-soa-gateway-api/pull/140 is merged.